conduct.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Code of conduct
+layout: default
+---
+
+# Code of conduct
+
+---
+
+In the interest of fostering an open and welcoming environment, we as
+contributors and maintainers pledge to making participation in our project and
+our community a harassment-free experience for everyone, regardless of age, body
+size, disability, ethnicity, gender identity and expression, level of experience,
+nationality, personal appearance, race, religion, or sexual identity and
+orientation.
+
+This Code of Conduct applies both within project spaces and in public spaces
```

Comment:
Project = IRC/Github; Public = anywhere else as long as they're publicly representing the project.